Formula/gobject-introspection.rb

```diff
@@ -22,18 +23,26 @@ class GobjectIntrospection < Formula
         :revision => "499ac89f8a9ad17d250e907f74912159ea216416"
   end
 
+  # submitted upstream as https://gitlab.gnome.org/GNOME/gobject-introspection/merge_requests/177
+  patch :DATA
```

Comment:
At this point are you de facto Gnome maintainer for macOS? 😆

Reply:
Just doing what I can... Most people there don't have any love for macOS 😢

Reply:
You're doing a really great job!